webofneeds/won-node/src/main/java/won/node/camel/processor/general/ReferenceToUnreferencedMessageAddingProcessor.java

```diff
@@ -95,7 +97,7 @@ public WonMessage process(final WonMessage message) throws WonMessageProcessingE
     }
     //persist the message
     messageEventRepository.save(messageEventPlaceholders);
-    return new WonMessage(messageDataset);
+    return message;
```

Comment:
is it save to change the message object and return it or should the changes be made on a that is returned copy?